client/core/device.go

```diff
@@ -155,16 +155,16 @@ func (d *Device) getConn(addr string) (c *coap.ClientCloseHandler, ok bool) {
 	return nil, false
 }
 
-func (d *Device) dial(ctx context.Context, addr net.Addr, dialOptions ...coap.DialOptionFunc) (*coap.ClientCloseHandler, error) {
+func (d *Device) dial(ctx context.Context, addr net.Addr) (*coap.ClientCloseHandler, error) {
```

Comment:
`dialOptions ...coap.DialOptionFunc` why it was removed?

Reply:
internal function called only from https://github.com/plgd-dev/device/blob/13db9d33afecf440fe157f76e7d0b1e06494c8ae/client/core/device.go#L183 so the parameter pack was never used